interfaces/console/src/app/console/subscribers/page.tsx

```diff
@@ -592,12 +572,56 @@ const Page = () => {
               },
             },
           });
+
+          await new Promise((resolve) => setTimeout(resolve, 3000));
+
+          const result = await getPackagesForSim({
+            variables: {
+              data: {
+                sim_id: plan.simId,
+              },
+            },
+          });
+
+          const packages = result?.data?.getPackagesForSim?.packages || [];
+
+          const mostRecentInactivePackage = packages
+            .filter((pkg) => pkg.is_active === false)
+            .sort(
+              (a, b) =>
+                new Date(b.start_date).getTime() -
+                new Date(a.start_date).getTime(),
+            )[0];
+
+          if (mostRecentInactivePackage) {
+            try {
+              await activatePackageSim({
+                variables: {
+                  data: {
+                    sim_id: plan.simId,
+                    package_id: mostRecentInactivePackage.id,
+                  },
+                },
+              });
+            } catch (error) {
+              console.error(
+                `Error activating package ${mostRecentInactivePackage.id}:`,
+                error,
+              );
+              throw error;
+            }
```

Comment:
Package active behaviour is being moved to backend side now, Since we have Queue packages behaviour now. Please discuss this with Elvis, he's implementing it on backend side.